docs/apache-airflow/authoring-and-scheduling/dynamic-task-mapping.rst

```diff
@@ -53,7 +53,7 @@ The grid view also provides visibility into your mapped tasks in the details pan
 
     In the above example, ``values`` received by ``sum_it`` is an aggregation of all values returned by each mapped instance of ``add_one``. However, since it is impossible to know how many instances of ``add_one`` we will have in advance, ``values`` is not a normal list, but a "lazy sequence" that retrieves each individual value only when asked. Therefore, if you run ``print(values)`` directly, you would get something like this::
 
-        LazyXComAccess(dag_id='simple_mapping', run_id='test_run', task_id='add_one')
+        [... 15 items ...]
```

Comment:
Curious what this means

Reply:
This is what will appear in the logs. Suggestions are welcome, but as mentioned in a thread above I don’t think it’s a good idea to print the entire list here.